packages/backend/src/db/house/houseStore.service.ts

```diff
@@ -0,0 +1,46 @@
+import { Injectable } from '@nestjs/common';
+import { InjectModel } from '@nestjs/mongoose';
+import { Model } from 'mongoose';
+import { House, HouseDocument, HouseModel } from './house.schema';
+
+@Injectable()
+export class HouseStoreService {
+  constructor(
+    @InjectModel(House.name)
+    private readonly HouseModel: Model<HouseDocument>,
+  ) {}
+
+  async create(createdHouseModel: HouseModel): Promise<House> {
+    const createdHouse = await this.HouseModel.create(createdHouseModel);
+    return createdHouse;
```

Comment:
please remove the await here. You can just return without saving to a variable first
```suggestion
    return this.HouseModel.create(createdHouseModel);
```